recommender/recommender.py

```diff
@@ -27,12 +27,37 @@ def emit(param1, param2):
 from mako.lookup import TemplateLookup
 
 from xblock.core import XBlock
-from xblock.fields import Scope, List, Dict
+from xblock.fields import Scope, List, Dict, Boolean, String
 from xblock.fragment import Fragment
 from xblock.reference.plugins import Filesystem
 
 from webob.response import Response
 
+
+def stem_url(url):
```

Comment:
I was curious. urlobject is buggy: 

    # URLObject("http://en.wikipedia.org/wiki/History_of_Ghana%23Growth_of_opposition_to_Nkrumah").without_fragment()
    > URLObject(u'http://en.wikipedia.org/wiki/History_of_Ghana%23Growth_of_opposition_to_Nkrumah')